opensrp-chw/src/kk/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java

```diff
@@ -82,12 +99,14 @@ protected void evaluateToddlerDangerSign(Map<String, ServiceWrapper> serviceWrap
 
         ToddlerDangerSignAction helper = new ToddlerDangerSignAction(context, alert);
 
-        Map<String, List<VisitDetail>> details = getDetails(Constants.EventType.CHILD_HOME_VISIT);
+        Map<String, List<VisitDetail>> details = getDetails(KKCoreConstants.ChildVisitEvents.TODDLER_DANGER_SIGN);
 
         BaseAncHomeVisitAction toddler_ds_action = new BaseAncHomeVisitAction.Builder(context, context.getString(R.string.toddler_danger_sign_month, serviceIteration))
                 .withOptional(false)
                 .withDetails(details)
                 .withFormName("child_hv_toddler_danger_sign")
+                .withPayloadType(BaseAncHomeVisitAction.PayloadType.SERVICE)
+                .withProcessingMode(BaseAncHomeVisitAction.ProcessingMode.SEPARATE)
```

Comment:
@issyzac - Here we need to agree whether we are doing ProcessingMode = SEPARATE or COMBINED. But based on the data conversation the other day, we might need this to be COMBINED. Combined treat the the fields submissions from this form as obs for the CHILD HOME VISIT EVENT.

Reply:
Sure thing, this is well noted!